getting-started/index.html

```diff
@@ -713,7 +713,8 @@ <h2><strong data-i18n="dgc-getting-started-paper-4-0">4. Print and fold your wal
 				<h4 data-i18n="dgc-community-modal-title" class="modal-title dogefont" id="community-modal-lable">COMMUNITY:</h4>
 				<h5 data-i18n="dgc-community-modal-get-involved">Get involved with Dogecoin! <br/><br/><span data-i18n="dgc-community-modal-participate">Participate in our Reddit community, chat on our IRC (Internet Chat) channel, or have fun on our forums.</span></h5>
 				<ul>
-					<li><a data-i18n="dogecoincom-reddit" href="http://www.reddit.com/r/dogecoin" target="_blank">Reddit Community</a><span data-i18n="dgc-community-modal-reddit-desc"> - The Main Hotspot of the Dogecoin Community.</span></li>
+					<li><a data-i18n="dogecoincom-reddit" href="https://www.reddit.com/r/dogecoin/" target="_blank">Reddit Community</a><span data-i18n="dgc-community-modal-reddit-desc"> - The Main Hotspot of the Dogecoin Community.</span></li>
+					<li><a data-i18n="dogecoincom-twitter" href="https://twitter.com/dogecoin" target="_blank">Twitter Community</a><span data-i18n="dgc-community-modal-twitter-desc"> - The Official Dogecoin Twitter Feed.</span></li>
```

Comment:
"Twitter" instead of "Twitter Community"

Reply:
```suggestion
					<li><a data-i18n="dogecoincom-twitter" href="https://twitter.com/dogecoin" target="_blank">Twitter</a><span data-i18n="dgc-community-modal-twitter-desc"> - The Official Dogecoin Twitter Feed.</span></li>
```